gateway/src/context.rs

```diff
@@ -39,10 +39,12 @@ impl ServerContext {
             OnceLock::new()
         };
 
-        const START_LATENCY_DECADE: i16 = -6;
-        const END_LATENCY_DECADE: i16 = 3;
+        // Track from 1 microsecond == 1e3 nanoseconds
+        const START_LATENCY_DECADE: u16 = 3;
+        // To 1000s == 1e9 * 1e3 == 1e12 nanoseconds
+        const END_LATENCY_DECADE: u16 = 12;
```

Comment:
nitpicky: should these be renamed from "decade" to "power" to match the argument names in the constructor?

Reply:
Yeah, good idea.

Reply:
Done in bc6f0d010